pyramid_zipkin/zipkin.py

```diff
@@ -199,3 +199,38 @@ def __exit__(self, _exc_type, _exc_value, _exc_traceback):
             binary_annotations=self.binary_annotations,
             span_id=self.span_id,
         )
+
+
+def zipkin_span(
+    service_name,
+    span_name,
+    binary_annotations=None,
+):
+    """Decorator utility for logging a function as a Zipkin span.
+    Note that the decorator doesn't accept timestamp annotations.
+    Any timestamps included in the decorator would likely be way off,
```

Comment:
It took me  a while to understand this. The idea is that the ZipkinContext context manager will handle timestamps for you, so don't bother passing them in, right?